ui/src/components/PFNotchedOutline.js

```diff
@@ -24,12 +24,12 @@ import React, {Component} from 'react';
  * pretty sparse and b) I'm not sure Material supports using their
  * NotchedOutline with anything other than TextField and Select components.
  */
-const PFNotchedOutline = () => (
+const PFNotchedOutline = (props) => (
```

Comment:
Parameter type?

Reply:
From what I can tell from other React code, it's just an object (props, along with state, is kinda a React thing; we don't create it ourselves). Added a param comment.